### ISSUE TITLE
**Three** build systems for **one** project

### ISSUE THREAD

**Issue by v217** [facebookexperimental/reason-native#192]
I am sure there are plenty of technical reasons. But if you are mostly not using javascript, it's just cumbersome  in order to build a native react program to install three build systems npm, esy and dune.
Even just the two build systems esy and dune mean a high learning curve for beginners.

**Comment by kyldvs** [comment]
Thanks for the issue! Are there suggestions/alternatives you would want to see us use or are you just asking to simplify this in general?

**Comment by v217** [comment]
From a beginner's point of view, who wants to get started with native reason, it's just that a better syntax for an existing ecosystem, so why not stick with dune/opam? As soon as one wants to use an existing opam package one has to get familiar with these tools anyway. I am just getting started so I guess this is really the perspective of a beginner, and not somebody who is familiar with the details. So yes simplify in general would help people get started.

**Comment by bandersongit** [comment]
Just as a point of clarification, esy works with opam packages out of the box, so familiarity with opam isn't needed unless one wishes to publish something to opam.

**Comment by v217** [comment]
OK Thanks, I guess for a beginner it's just confusing.
On the reasonml site there is yet another workflow
https://reasonml.github.io/docs/en/native
So it would be easier if there is like in python just one option if you don't have an special wishes or you are an expert who knows what he is doing.

**Comment by kyldvs** [comment]
I agree this isn't the most simple setup and we can work to make it easier for beginners. We can clarify in documentation best practices, and there are also project like https://github.com/esy/pesy that will be able to help with this.

Thanks for the perspective, I'll keep this open to track improving the setup.
